korge/src/commonMain/kotlin/com/soywiz/korge/util/Observable.kt

```diff
@@ -0,0 +1,17 @@
+package com.soywiz.korge.util
+
+import kotlin.reflect.*
+
+class Observable<T>(val initial: T, val before: (T) -> Unit = {}, val after: (T) -> Unit = {}) {
```

Comment:
Yep, kotlin delegates doesn't fit here. I need to execute some code before the variable has changed, but I don't need to prevent the change, I just need to prevent executing this code. So yes, I will propose a PR to kds with this class.

Reply:
I'll update this PR once the new version of kds with https://github.com/korlibs/kds/pull/17 is released